credentials/academic-credential/credential.json

```diff
@@ -0,0 +1,68 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3c-ccg.github.io/vc-ed/plugfest-1-2022/jff-vc-edu-plugfest-1-context.json",
```

Comment:
```suggestion
```

You can remove that context as the terms are in the other context files. You can test it out in the [JSON-LD Playground](https://json-ld.org/playground/) by watching (or diffing/comparing) the N-Quads output before and after this context URL is removed. Removing it doesn't change the output, so that context isn't providing anything now that OB3 has these terms.

Hope that helps!